### ISSUE TITLE
VAMC Billing & Insurance: adopt V1 and V3 components where components are not used

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#16962]
## Description
VAMC Billing & Insurance pages need the following updates to components:

Example pages:

- [Boston Billing and Insurance](https://www.va.gov/boston-health-care/billing-and-insurance/)
- [Minneapolis Billing and Insurance](https://www.va.gov/minneapolis-health-care/billing-and-insurance/)

### Elements that are not using components that should be (Backlog)

1. ~Telephone~
   - ~The first phone number, for copay questions, is not using the component but should be using [the v1](https://design.va.gov/components/telephone)~
   - ~All other phone numbers are using the correct component~
   - ~We should confirm that Exts in phone numbers within the component are parsed correctly~
2. ~Link~
   - ~Facility names should use the link component~
3. [Back to top component](https://design.va.gov/components/back-to-top)
   - Update 5/3: After determining that this component does not actually have built in intelligence to only show based on page height:
      -  This should still be added, the Top task pages are on the longer side so having this on these pages will probably make sense more often than not. 

## Acceptance Criteria
- [x] Back to top component has been added
- [ ] Update and run existing test(s).
- [ ] If additional tests are needed, stub a ticket to note this for future work.
- [ ] Requires accessibility review

**Comment by maxx1128** [comment]
I estimate this ticket at a **3** for moving over a good amount of relatively simple elements over to components (deja vu)

**Comment by laflannery** [comment]
I removed the link component from this ticket, after working with PW, the link I noted on this page is CMS content and we are limited with what we can do with that type of content

**Comment by eselkin** [comment]
2, similar to Records ticket

**Comment by laflannery** [comment]
@eselkin You mentioned in scrum you merged this, so I just checked and it on Prod, excellent. But for bookeeping purposes - I can't see a PR mentioned anywhere. Is that OK, not OK, should there be?

**Comment by eselkin** [comment]
@laflannery Of course it's because I linked the zenhub ticket not the github ticket. Let me fix it https://github.com/department-of-veterans-affairs/content-build/pull/2064

**Comment by laflannery** [comment]
Closing as Eli and I have both verified on Prod
